docs/en/observability/synthetics-quickstart-fleet.asciidoc

```diff
@@ -0,0 +1,93 @@
+image::images/synthetics-agent-policy.png[Synthetics integration]
+
+[discrete]
+[[synthetics-quickstart-fleet-step-four]]
+== Step 4: Add an {agent} to {fleet}
+
+{agent} is a single, unified agent that you can deploy to hosts or containers to collect data and send it to the {stack}. Behind the scenes, {agent} runs {beats} or {elastic-endpoint} required for your configuration. 
+
+NOTE: To run synthetics test, you must use the *elastic-agent-complete* Docker container, which contains the dependencies to run synthetic monitors.
+
+To add an {agent} to {fleet}, you'll need an enrollment token. You can use the default enrollment token for your policy or create new enrollment tokens as needed. To learn more, see https://www.elastic.co/guide/en/fleet/current/fleet-enrollment-tokens.html[{fleet} enrollment tokens].
+
+To add the *elastic-agent-complete* to {fleet}, run the following Docker command. Ensure to substitute the correct `FLEET_URL` and `FLEET_ENROLLMENT_TOKEN`.
+
+// NOTE: We do NOT use <1> references in the below example, because they create whitespace after the trailing \
+// when copied into a shell, which creates mysterious errors when copy and pasting!
+[source,sh,subs="attributes"]
+----
+docker run \
+  --rm \
+  --name=synthetics \
+  --user=elastic-agent \
+  -e FLEET_URL=http://localhost:8220 \
+  -e FLEET_ENROLLMENT_TOKEN=enrollment-token \
+  -e FLEET_ENROLL=1 \
+  {synthetics-image} -e \
+----
+
+NOTE: You may need to set additional environment variables. See the https://www.elastic.co/guide/en/fleet/current/agent-environment-variables.html[{agent} environment variables guide] for more information. See https://www.elastic.co/guide/en/fleet/master/elastic-agent-container.html[Run {agent} in a container] for more information on running {agent} with Docker.
+
+[[synthetics-quickstart-fleet-step-five]]
+== Step 5: Configure an Elastic Synthetics integration policy beta[]
+
+Elastic Synthetics tests can be configured through {fleet} using the Elastic Synthetics integration. To learn more, see https://www.elastic.co/guide/en/fleet/current/fleet-quick-start.html#add-synthetics-integration[Elastic Synthetics].
+
+[role="screenshot"]
+image::images/synthetics-integration.png[Synthetics integration]
+
+In the example above, our library of synthetic tests is downloaded from the
+remote zip endpoint for https://github.com/elastic/synthetics-demo/tree/main/todos/synthetics-tests[our todos example]. The *zip URL* refers to the endpoint where the test project exists. *Folder* refers to the relative path where the synthetic journey files are located. {heartbeat} will invoke the synthetics library on this folder. *zip URL username* and *zip URL password* are blank in this example, but if provided, they will be sent as HTTP Basic Authentication headers to the remote zip endpoint. *Params* refers to dynamic values that always need to be defined for our tests to run. This is often helpful switching between production and development values. 
```

Comment:
I think there's a word missing here. Probably should say, "This is often helpful when switching between..."